projects/components/src/header/page/page-header.component.ts

```diff
@@ -99,6 +104,9 @@ export class PageHeaderComponent implements OnInit {
   @Input()
   public hidePageTimeRange?: boolean = false;
 
+  @Input()
+  public showRefreshButton?: boolean = false;
```

Comment:
this feels off - both because other boolean inputs are hideXyz (so this is inverted), but also because not all pairings of the two booleans make sense. This is only checked if we're hiding the time range.

Reply:
Updated.

Reply:
Addressed the inversion, but not the fact that 2 booleans introduces 4 states when only 3 are actually meaningful. Maybe combine these two booleans into a mode enum (would need to handle compatibility, but that's straightforward by deprecating boolean but using to set mode default if that's unset).

Reply:
Makes sense. Will update.